docs/index.rst

```diff
@@ -268,86 +206,76 @@ Topic Management
+3. Alternatively, run KSQL CLI to get to the KSQL CLI prompt.
 
-5. **DEVELOPMENT -> KSQL -> PERSISTENT QUERIES**: View the existing KSQL queries, which are continuously running. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW QUERIES;``).
+   .. sourcecode:: bash
 
-     .. figure:: images/ksql_queries_list.png
-        :alt: image
+        docker-compose exec ksql-cli ksql http://ksql-server:8088
 
-6. **DEVELOPMENT -> KSQL -> STREAMS**: View messages from different KSQL streams and tables. Right click on your stream of choice, select ``Query`` which takes you to the Query Editor with a pre-populated query such as ``select * from WIKIPEDIA limit 5;``.  Click on the ``Run query`` button to run.
+4. View the existing KSQL streams. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW STREAMS;``).
 
-     .. figure:: images/ksql_query_topic.png
-        :alt: image
+   .. figure:: images/ksql_streams_list.png
+      :alt: image
 
-7. **DEVELOPMENT -> KSQL -> STREAMS**: Create a new stream from an existing topic. Click on the button ``Create Stream`` and follow the prompts.
+5. Describe the schema (fields or columns) and source and sink of an existing KSQL stream. Click on ``WIKIPEDIA``.
 
-8. **DEVELOPMENT -> KSQL -> QUERY EDITOR**: View the configured KSQL server properties set in the docker-compose.yml file. In the query editor, type ``SHOW PROPERTIES;`` and then click on the ``Run query`` button.
+   .. figure:: images/wikipedia_describe.png
+      :alt: image
 
-     .. figure:: images/ksql_properties.png
-        :alt: image
+6. View the existing KSQL tables. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW TABLES;``).
 
-9. In this demo, KSQL is run with Confluent Monitoring Interceptors configured which enables |c3| Data Streams to monitor KSQL queries. The consumer group names ``ksql_query_`` correlate to the KSQL query names above, and |c3| is showing the records that are incoming to each query.
+   .. figure:: images/ksql_tables_list.png
+      :alt: image
 
-* View throughput and latency of the incoming records for the persistent KSQL "Create Stream As Select" query ``CSAS_WIKIPEDIABOT``, which is displayed as ``ksql_query_CSAS_WIKIPEDIABOT`` in |c3|.
+7. View the existing KSQL queries, which are continuously running. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW QUERIES;``).
 
-   .. figure:: images/ksql_query_CSAS_WIKIPEDIABOT.png
+   .. figure:: images/ksql_queries_list.png
       :alt: image
 
-* View throughput and latency of the incoming records for the persistent KSQL "Create Table As Select" query ``CTAS_EN_WIKIPEDIA_GT_1``, which is displayed as ``ksql_query_CTAS_EN_WIKIPEDIA_GT_1`` in |c3|.
+8. View messages from different KSQL streams and tables. Right click on your stream of choice, select ``Query`` which takes you to the Query Editor with a pre-populated query such as ``select * from WIKIPEDIA;`` and it will show results for newly arriving data.
```

Comment:
Right clicking on my browser (Firefox 68.0.1) doesn't work (it shows the firefox context menu).  Instead, I can see these options by clicking the elipse (`...`) button C3 presents me.

Reply:
@rspurgeon good catch.  This behavior has probably changed in 5.3.0, so now it's just a "normal" click.  Updated.